pyiron/lammps/control.py

```diff
@@ -201,42 +201,57 @@ def pressure_to_lammps(self, pressure, rotation_matrix):
         Lammps handles complex cells in a particular way, namely by using an upper triangular cell. This means we may
         need to convert our pressure tensor to a new coordinate frame. We also handle that transformation here.
 
+        I case of a single pressure value, it is again returned as a single pressure value, to be used with the "iso"
+        option (i.e., coupled deformation in x, y, and z).
+
         Finally, we also ensure that the units are converted from pyiron's GPa to whatever Lammps needs.
 
         Args:
             pressure (float/list/tuple/numpy.ndarray): The pressure(s) to convert.
             rotation_matrix (numpy.ndarray): The 3x3 matrix rotating from the pyiron to Lammps coordinate frame.
 
         Returns:
-            (numpy.ndarray): pxx, pyy, pzz, pxy, pxz, and pyz to be passed to Lammps.
+            (list): pxx, pyy, pzz, pxy, pxz, and pyz to be passed to Lammps.
+
+            or
+
+            (float): a single, isotropic pressure to be used with the "iso" option
```

Comment:
Is this the right way to document different return values?

Reply:
Looks reasonable to me, but I would direct you to the google docstring style guide. I don't think I've seen the case of 'or' for different return types there before, but I'm ok with how you have it here.